<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QMainWindow" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scrum pinboard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10">
    <property name="spacing">
     <number>15</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,1,1,1">
      <item>
       <widget class="QLabel" name="nameLabel">
        <property name="font">
         <font>
          <family>GOST type B</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="newProjectPushButton">
        <property name="font">
         <font>
          <family>GOST type B</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Новый проект</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="profilePushButton">
        <property name="font">
         <font>
          <family>GOST type B</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Профиль</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exitPushButton">
        <property name="font">
         <font>
          <family>GOST type B</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Выйти</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="projectsScrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="projectsScrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>776</width>
         <height>506</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="projectsScrollAreaWidgetContentsVerticalLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>newProjectPushButton</tabstop>
  <tabstop>profilePushButton</tabstop>
  <tabstop>exitPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
